### PR TITLE
Fixed for issue #2767.

### PR DESCRIPTION
**Pull request by JordanGS**
Addresses and fixed issue: https://github.com/zaproxy/zaproxy/issues/2767

### CODE REVIEW

File: src/org/zaproxy/zap/extension/alert/ExtensionAlert.java [hunk 1]
```diff
@@ -603,9 +603,12 @@ void recalcAlerts() {
                 int alertId = v.get(i).intValue();
                 RecordAlert recAlert = tableAlert.read(alertId);
                 Alert alert = new Alert(recAlert);
-                if (!allAlerts.contains(alert)) {
-                    allAlerts.add(alert);
-                }
+				if (alert.getHistoryRef() != null) {
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/alert/ExtensionAlert.java]: It could still use spaces for indentation, as surrounding code.

**Comment by thc202** [comment]
Could you change the commit message or description of pull request to reference that it addresses the issue? [1]

[1] https://help.github.com/articles/closing-issues-via-commit-messages/

File: src/org/zaproxy/zap/extension/alert/ExtensionAlert.java (hunk 1 above)
Comment (JordanGS) [inline src/org/zaproxy/zap/extension/alert/ExtensionAlert.java]: @thc202 I'm not sure why that happened, i'm looking at my file right now and the `if` is in line and properly indented :S

**Comment by thc202** [comment]
Superseded by #2769.

**Pull request closed without merging**